pages/linux/mklost+found.md

```diff
@@ -0,0 +1,8 @@
+# mklost+found
+
+> Create a lost+found directory.
+> More information: <https://manned.org/mklost+found>.
+
+- Create a lost+found directory:
```

Comment:
@CleanMachine1 as I understand it, `fsck` only creates a `lost+found` folder in the root directory, but this command creates one in the current directory. I don't know what that actually means, but I think we should include that.
```suggestion
- Create a `lost+found` directory in the current directory:
```
(Also I think the backticks make sense here)

Reply:
Ok, just for reference, the lost+found is what fsck outputs, which would normally be corrupted, damaged files which is normally probably junk data.

However I've never used it, I just saw it when making my file system when tabbing